plugins/SkyCultureMaker/src/ScmSkyCulture.cpp

```diff
@@ -60,11 +59,9 @@ void scm::ScmSkyCulture::removeConstellation(const QString &id)
 
 scm::ScmConstellation *scm::ScmSkyCulture::getConstellation(const QString &id)
 {
-	for (auto &constellation : constellations)
-	{
-		if (constellation->getId() == id) return constellation.get();
-	}
-	return nullptr;
+	auto it = std::find_if(constellations.begin(), constellations.end(),
```

Comment:
Funny, although this is following the "best practices", it (together with the subsequent `return` line) does look more complicated than the naive for loop, and I think is not worth it.